mappings/net/minecraft/util/io/CompoundTag.mapping

```diff
@@ -52,7 +52,7 @@ CLASS nu net/minecraft/util/io/CompoundTag
 		ARG 1 key
 	METHOD j getByteArray (Ljava/lang/String;)[B
 		ARG 1 key
-	METHOD k getCompoundTag (Ljava/lang/String;)Lnu;
+	METHOD k getCompound (Ljava/lang/String;)Lnu;
```

Comment:
ok.. I'm interested as to why you made this change

Reply:
Fixed.